libbeat/cmd/instance/ilm.go

```diff
@@ -0,0 +1,202 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package instance
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
+)
+
+type ilmConfig struct {
+	RolloverAlias string `config:"ilm.rollover_alias" `
+	Pattern       string `config:"ilm.pattern"`
+}
+
+var ILMPolicy = common.MapStr{
+	"policy": common.MapStr{
+		"type": "timeseries",
+		"phases": common.MapStr{
+			"hot": common.MapStr{
+				"actions": common.MapStr{
+					"rollover": common.MapStr{
+						"max_size": "25gb",
+						"max_age":  "30d",
+					},
+				},
+			},
+		},
+	},
+}
```

Comment:
Could you move this to a json file so that users can easily overwrite and adapt to their needs?

Reply:
I had it a first as a json file but remove the approach on purpose for now: https://github.com/elastic/beats/pull/7963#discussion_r214820033 Happy to continue the discussion.